CRM/Logging/Schema.php

```diff
@@ -369,8 +369,10 @@ public function fixSchemaDifferencesForAll($rebuildTrigger = FALSE) {
     }
 
     if ($rebuildTrigger) {
-      // invoke the meta trigger creation call
-      CRM_Core_DAO::triggerRebuild($table);
+      foreach ($diffs as $table => $cols) {
+        // invoke the meta trigger creation call
+        CRM_Core_DAO::triggerRebuild($table);
```

Comment:
@ejegg - agree that it seems weird to only rebuild the last in the list.

Just verbalizing more to see if I understand... The `triggerRebuild()` function can be called with a specific table-name or with NULL (for all tables).  From the title of the PR, I would have expected it to rebuild all tables, e.g.:

```php
CRM_Core_DAO::triggerRebuild(); // default to $tableName==NULL; rebuild all tables
```

It sounds like the current PR will call `triggerRebuild($table)` for a selected list of tables (`$diffs`) -- e.g. skipping triggers for new log tables (`empty($this->logs[$table])`) but rebuilding triggers for existing log tables (**not** `empty($this->logs[$table])`).

This is fairly particular behavior, and I'm not familiar with the situation that motivated this patch. Just wondering if it makes sense to you.